stages/org.osbuild.selinux

```diff
@@ -4,26 +4,33 @@ import pathlib
 import sys
 
 import osbuild.api
-from osbuild.util import selinux
+from osbuild.util import parsing, selinux
 
 
-def main(tree, options):
+def main(args):
+    # Get the path where the tree is
+    options = args["options"]
     file_contexts = options.get("file_contexts")
     exclude_paths = options.get("exclude_paths")
+    target = options.get("target", "tree:///")
+    root, target = parsing.parse_location_into_parts(target, args)
 
     if file_contexts:
-        file_contexts = os.path.join(f"{tree}", options["file_contexts"])
+        if "://" not in file_contexts:
+            file_contexts = os.path.normpath(f"{args["tree"]}/{file_contexts}")
+        else:
+            file_contexts = parsing.parse_location(file_contexts, args)
         if exclude_paths:
-            exclude_paths = [os.path.join(tree, p.lstrip("/")) for p in exclude_paths]
-        selinux.setfiles(file_contexts, os.fspath(tree), "", exclude_paths=exclude_paths)
+            exclude_paths = [os.path.normpath(f"{root}/{target}/{p}") for p in exclude_paths]
+        selinux.setfiles(file_contexts, os.path.normpath(root), target, exclude_paths=exclude_paths)
 
     labels = options.get("labels", {})
     for path, label in labels.items():
-        fullpath = os.path.join(tree, path.lstrip("/"))
+        fullpath = parsing.parse_location(path, args)
         selinux.setfilecon(fullpath, label)
 
     if options.get("force_autorelabel", False):
-        stamp = pathlib.Path(tree, ".autorelabel")
+        stamp = pathlib.Path(root, ".autorelabel")
```

Comment:
Even though this option is marked "Do not use", perhaps we should mention in its description (in the schema) that it will now affect whichever tree (regular stage tree or mount root) is mentioned in the `target` option.